tests/playwright.config.ts

```diff
@@ -0,0 +1,17 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  expect: { timeout: 5000 },
+  forbidOnly: !!process.env.CI,
+  fullyParallel: false,
```

Comment:
Don't we want this to be true?

Reply:
> Don't we want this to be true?

When it's true the tests are mad flaky

Reply:
Gotcha. We can fix that in another PR I guess if we want to speed it up.